### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by matejvasek**
* Fixed typos
* Removed trailing whitespaces
* Added EoLs at EoFs

### CODE REVIEW

File: cmd/environment.go [hunk 1]
```diff
@@ -37,8 +37,7 @@ DESCRIPTION
 	the version of func, the version of the function spec, the default builder,
 	available runtimes, and available templates.
 `,
-		SuggestFor: []string{"env", "environemtn", "enviroment", "enviornment", "enviroment"},
```
Comment (matejvasek) [inline cmd/environment.go:40]: This typo-detect-hint works out-of-the box.

File: cmd/templates.go [hunk 2]
```diff
@@ -56,11 +56,6 @@ EXAMPLES
 	o Return Go templates in a specific repository
 		$ {{rootCmdUse}} templates go --repository=https://github.com/boson-project/templates
 `,
-		SuggestFor: []string{"template", "templtaes", "templatse", "remplates",
```
Comment (matejvasek) [inline cmd/templates.go:59]: This typo-detect-hint works out-of-the box.

**Comment by matejvasek** [comment]
PTAL @gauron99 @lkingland @matzew

File: CHANGELOG.md [hunk 3]
```diff
@@ -590,7 +590,7 @@ All notable changes to this project will be documented in this file. See [standa
 ### Docs
 - fix function typos
 - setting up remote access to kind clusters
-- wireguard configuraiton for OS X
+- wireguard configuration for OS X
```
Comment (matejvasek) [inline CHANGELOG.md:593]: I wonder what's the deal with wireguard. What's is doing here?

Comment (matejvasek) [reply CHANGELOG.md:593]: I mean it's unrelated to this review. Still...

Comment (matejvasek) [reply CHANGELOG.md:593]: @lkingland any idea what it was there?

Comment (lkingland) [reply CHANGELOG.md:593]: This was related to an old example which is no longer used

**Review by lkingland — APPROVED** [review]

**Pull request merged**